### PR TITLE
Add: Added JavaScript example for using FirefoxOptions

### PR DESCRIPTION
**Pull request by natanportilho**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Added JavaScript code example for using FirefoxOptions

### Motivation and Context
Making Selenium Friendlier for JS developers.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md [hunk 1]
```diff
@@ -36,7 +36,14 @@ var driver = new FirefoxDriver(options);
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+import * as firefox from 'selenium-webdriver/firefox';
+import { Builder } from 'selenium-webdriver';
+
+const options = new firefox.Options();
```
Comment (Harsha509) [inline docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md]: hi @natanportilho,

Like other code samples, can you set any firefox capability in the JS code sample

adding this line will do i think =>  "options.headless();"

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md [hunk 2]
```diff
@@ -36,7 +36,14 @@ var driver = new FirefoxDriver(options);
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+import * as firefox from 'selenium-webdriver/firefox';
```
Comment (Harsha509) [inline docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md]: Hi @natanportilho,

i was unable to test the code sample, i faced the below issue

import * as firefox from 'selenium-webdriver/firefox';
       ^

SyntaxError: Unexpected token *
    at Module._compile (internal/modules/cjs/loader.js:723:23)

but placing below line worked for me

const firefox = require('selenium-webdriver/firefox');

Comment (natanportilho) [reply docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md]: Hi @Harsha509,

Thanks for your review, I've applied these changes.

File: docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md (hunk 1 above)
Comment (natanportilho) [reply docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md]: Hello, I've changed the code to use the options this way. Thank for your help.

**Pull request merged**